ObjectCreation/G3d.FCMacro

```diff
@@ -0,0 +1,475 @@
+# -*- coding: utf-8 -*-
+# (c) 18Turbo, 2022
+
+__Name__ = 'Generador3D'
+__Comment__ = 'Create a solid with 2 or 3 sketches (views)'
+__Author__ = '18Turbo'
+__Version__ = '0.1.0'
+__Date__ = '2022-09-05'
+__License__ = 'LGPL-2.0-or-later'
+__Web__ = 'https://github.com/18turbo/Generador3DFreeCAD'
+__Wiki__ = 'https://github.com/18turbo/Generador3DFreeCAD/wiki'
+__Icon__ = 'https://github.com/18turbo/Generador3DFreeCAD/blob/main/generador3D.svg'
+__Help__ = ''
+__Status__ = 'beta'
+__Requires__ = 'FreeCAD >= v0.20'
+__Contact__ = 'https://github.com/18turbo'
+__Communication__ = 'https://github.com/18turbo/Generador3DFreeCAD/issues'
+__Files__ = 'g3d/icons/g3d.svg,g3d/ui/g3d15.ui'
+
+
+# (En español:)
+# Funcionalidad:    Genera una pieza 3D a partir de 2 o 3 sketch, cada uno en una vista distinta (planta, frontal o lateral)
+#
+# Descripción:      Una vez ejecutada la macro, habrá que asociar cada sketch a cada vista, y pulsar el botón de OK.
+#
+#
+# Notas de versiones:
+# v.0.1.0:      Adaptación del código para varios idiomas
+#               Corrección de error en la deselección de vistas
+#               Cambio de nombres de ficheros
+# v.0.0.16:     Corregido errores en la selección de vistas
+#               Evita el error de introducir la longitud menor o igual al offset
+# v.0.0.15:     Cambiada la interfaz
+#               Añadido Offset y Longitud a cada vista
+# v.0.0.14:     Adaptación al Addon Manager
+# v.0.0.13:     Depuración de código
+# v.0.0.12:     Depuración de código
+#               Posibilidad de crear una carpeta automáticamente donde meter las vistas y ocultarlas
+# v.0.0.11:     Incluida tolerancia 3D para evitar problemas en los límites.
+# v.0.0.10:     Corrección de errores en sketch negativos
+# v.0.0.9:      Cambio de interfaz y corrección de errores. Beta funcional.
+# v.0.0.8:      Corrección de errores
+# v.0.0.7:      Corrección de errores
+# v.0.0.6:      Primera versión BETA
+# v.0.0.5 ALFA: Genera la pieza 3D con 3 vistas
+# v.0.0.4 ALFA: Arregla la selección en las vistas en la pestaña Task (Tarea)
+#
+#
+# BetaTester: Rafael García Rodríguez
+# BetaTester: 18Turbo
+
+# Traducción inglés:    Rocco Vicedomini
+ 
+
+import FreeCAD,FreeCADGui,Part
```

Comment:
Please use
```
import FreeCAD as app
import FreeCADGui as gui
```
for consistency with other macros? Thanks!

Reply:
Done